chebi-update/src/main/java/org/reactome/release/chebiupdate/ChebiUpdater.java

```diff
@@ -416,12 +416,18 @@ private void updateReferenceEntities(GKInstance molecule, String chebiName, GKIn
 	 * Gets the Creator of some instance.
 	 * @param inst - the Instance to get the creator of.
 	 * @return A GKInstance. It is the value in the "author" attribute (most likely, it will be a Person object) of the InstanceEdit that is associated with "created" attribute of <code>inst</code>.
+	 * If the instance does not have a "created" attribute, then NULL will be returned.
 	 * @throws InvalidAttributeException
 	 * @throws Exception
 	 */
 	private static GKInstance getCreator(GKInstance inst) throws InvalidAttributeException, Exception
 	{
 		GKInstance createdInstanceEdit = (GKInstance) inst.getAttributeValue(ReactomeJavaConstants.created);
+		if (createdInstanceEdit == null)
+		{
+			logger.error("Instance {} does not have a value for \"created\" attribute!", inst.toString());
```

Comment:
Should this be a warn instead of an error?

Reply:
Yes, probably.